lib/ex_plasma/utxo.ex

```diff
@@ -229,4 +235,17 @@ defmodule ExPlasma.Utxo do
   defp truncate_leading_zero(<<0>>), do: <<0>>
   defp truncate_leading_zero(<<0>> <> binary), do: truncate_leading_zero(binary)
   defp truncate_leading_zero(binary), do: binary
+
+  # Validates that the Utxo is in the expected formats. Returns an error tuple
+  defp validate_output(%{owner: <<0::160>>}), do: {:error, {:output_guard, :cannot_be_zero}}
```

Comment:
Owner or output guard?

Reply:
@InoMurko yeah I should probably switch this back to `owner` for now. I'll open an issue to suggest we move wholesale to `output_guard` as that's what the contract uses to define this value.